nodeup/pkg/model/firewall.go

```diff
@@ -87,7 +87,7 @@ iptables -A FORWARD -w -p ICMP -j ACCEPT
 fi
 `
 	return &nodetasks.File{
-		Path:     "/home/kubernetes/bin/iptables-setup",
+		Path:     "/opt/kops/helpers/iptables-setup",
```

Comment:
We might have to create this directory explicitly, but I think it's a good idea to pick a more reasonable directory.

For immutable OSes, we might have to bind-mount these also, if we're seeing failures there.

I think we could also do `/opt/kops/bin/iptables-setup`

Reply:
I chose the "helpers" dir because that was the dir chosen for the Docker healthcheck.
Will give this a try with CoreOS also when this is merged and check again.
Changing to `/opt/kops/bin/iptables-setup` sounds good to me.